hack/common.sh

```diff
@@ -41,24 +41,29 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
   examples/deployment
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
-
 readonly OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
-readonly OS_CROSS_COMPILE_TARGETS=(
+
+if [ "_$@" = "_test/e2e/e2e.test" ]; then
```

Comment:
to clarify: I guess reusing the test loop is ok, or the other loop.  But i see these both as lateral moves ~ so if we really think one or the other is better, i'd ask for a concrete suggestion on which implementation we want.